.github/workflows/trivvy.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Trivy vulnerability scanner
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - devel
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install go
+        uses: actions/setup-go@v3.3.1
+        with:
+          go-version: ^1.18
+
+      - name: Build an image from Dockerfile
+        run: |
+          CONTAINER_CMD=docker make containerized-test TARGET=trivvy
```

Comment:
just use `make containerized-build` ?

Reply:
we can, so `devel` tag will be used in the scanner  @nixpanic